module/zfs/dmu_send.c

```diff
@@ -1760,6 +1759,7 @@ send_reader_thread(void *arg)
 	fstrans_cookie_t cookie = spl_fstrans_mark();
 	struct send_range *range = bqueue_dequeue(inq);
 	int err = 0;
+	uint64_t spill = 0;
```

Comment:
Looks like unneeded initialization. And may be it could be moved down to the only place where it is used.

Reply:
Originally, I put does `uint64_t spill = piggyback_unmodified_spill(srta, range);` under `case REDACT:` But some versions of compiler complained about can't but label on declaration.

Reply:
That is why `case PREVIOUSLY_REDACTED:` has curly brackets. I'm fine either way, but the way you've done it initialization with zero makes no sense.

As a third option you could move it inside the `while` loop, and then move `piggyback_unmodified_spill()` into separate `case OBJECT:` where it only makes sense, with fall-through to other cases after it.

Reply:
Ok, I go with third options.